cfpb/cfgov-refresh
# Moving adult fin-ed into cfgov-refresh

Moving adult fin-ed into cfgov-refresh

## Additions

- Added `cfgov/jinja2/v1/adult-financial-education/index.html`


## Changes

- Modified `cfgov/cfgov/urls.py` to remove the sheer related code.


## Testing

1. Visit cfgov/jinja2/v1/adult-financial-education/index.html and compare to production.
2. Use some tool, like https://www.npmjs.com/package/broken-link-checker to check the links.


## Checklist
